### ISSUE TITLE
All PHET ZIMs (but one) are stored twice in the library 

### ISSUE THREAD

**Issue by B-root74** [openzim/zim-requests#840]
It seems PHET ZIMs have been duplicated by mistake in the library.

For instance you have `phet_uk_all_2023-04.zim` and `phet_uk_2023-01.zim`. ZIM sizes and number of media / article seems almost identical.

Same apply for all languages you have.

The only exception is `phet_ab_2022-12.zim` which does have its "counterpart". But it looks like there is no (more ?) content in Abkhazian in PHET: https://phet.colorado.edu/en/simulations/filter?locale=ab

**Comment by Popolechien** [comment]
Yeah the names are different (notice the `_all_` before the date) and the system thus thinks they're separate.

@benoit74 worth cleaning up, no?

**Comment by Popolechien** [comment]
Yolo.

**Comment by benoit74** [comment]
Done, library will be updated in at most 30 minutes
